tests/Benchmark/SpanBench.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Tests\Benchmark;
+
+use PhpBench\Benchmark\Metadata\Annotations\Iterations;
+use PhpBench\Benchmark\Metadata\Annotations\Revs;
+use Sentry\Tracing\Span;
+use Sentry\Tracing\TransactionContext;
+
+class SpanBench
```

Comment:
Can this class be made `final` or is there some reason for not doing it (I never used `PhpBench`)?